etcd/lease_manager.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
Let's just put the contents of this file in lease.go. I don't see a strong reason to split them up right now.